Scoop/Model/Ride.swift

```diff
@@ -7,7 +7,7 @@
 
 import Foundation
 
-struct Ride: Codable {
+class Ride: Codable {
```

Comment:
Same as above. Wait I think we talked about this in Duffield but don't remember LOL

Reply:
Yep Yep 😆